app/src/main/java/woowacourse/shopping/constant/Format.kt

```
@@ -0,0 +1,5 @@
+package woowacourse.shopping.constant
+
+object Format {
+    fun formatPrice(price: Int): String = String.format("%,d원", price)
```

detekt (1.23.8)

[warning] String.format("%,d원", price) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)